src/internal/connector/sharepoint/site_pageable.go

```diff
@@ -0,0 +1,24 @@
+package sharepoint
+
+import (
+	kioser "github.com/microsoft/kiota-abstractions-go/serialization"
```

Comment:
So long as we're aliasing this import (I think I'd prefer not to, but we're already doing it), we should keep the alias standard.

```suggestion
	absser "github.com/microsoft/kiota-abstractions-go/serialization"
```